resources/dicts/events/lifegen_events/murder.json

```diff
@@ -307,25 +308,27 @@
-        "You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they ask you what the smell is with a wrinkled nose. You tell them not to worry about it and push them forward, saying that the treasure is inside. With an anxious look at you, they slowly head in. You don't stay to listen to the foxes' feasting.",
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, already having gone to StarClan.",
+        "You offer to take v_c onto a grand and exciting adventure out of the nursery. You help v_c sneak out and the two of you run into the woods. When you're far enough from camp, you suddenly speed up, leaving v_c behind. You loop back to camp, alone, and no one asks where you've been.",
+        "You tell v_c to follow you, saying that you want to show them a shiny treasure you found recently. They happily agree and the two of you sneak out of camp. You lead v_c to a foxes' den, and they ask you what the smell is with a wrinkled nose. You tell them not to worry about it and push them forward, saying that the treasure is inside. With an anxious look at you, they slowly head in, and never come back out.",
         "You ask v_c if they want to go on a secret mission with you, and they jump excitedly at the opportunity. You lead them out of camp and head towards the Thunderpath together. When you start to hear a monster in the distance, you pressure v_c into walking out on the path, saying that you'll be right behind them. As the monster zooms toward the two of you, you quickly run back to the other side of the Thunderpath while v_c is frozen in the headlights. You don't bother looking back.",
-        "You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it, saying that it's about time they had proper food instead of milk. They swallow it down with difficulty, and later you hear v_c's pained wails from the medicine den. Their body is cold and frothing at the mouth when you bury them."
+        "You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it, saying that it's about time they had proper food instead of milk. They swallow it down with difficulty, and they don't ever wake up again."
```

Comment:
this can be changed to match the previous one

"You bring v_c a piece of crow-food that you found by the side of the Thunderpath, claiming that it's from a special type of squirrel. They wrinkle their nose at it, but you encourage them to eat it. They don't wake up ever again."